### ISSUE TITLE
Release new version for agda 2.7

### ISSUE THREAD

**Issue by felixwellen** [agda/cubical#1148]
Agda 2.7 was released and we should release a new version of the cubical library, following the intructions here:

https://github.com/agda/cubical/blob/master/RELEASE.md

I recently used the library with the current master of agda, so I expect there is nothing else to do and anyone with the rights can do a release (and anyone who can make a PR can do the changes). I'll do this at some point after my vacation.

**Comment by foxyseta** [comment]
Yep, I'm currently still able to run `make test` on `main` (35d29193477fe168edfe4bff66af984068bf85e4) using Agda v2.7.0.1.
